### ISSUE TITLE
Gives an SVG error but works

### ISSUE THREAD

**Issue by warapitiya** [crisbeto/angular-svg-round-progressbar#35]
When it's the initial loading it gives 2 errors like

Error: Invalid value for <path> attribute d="M NaN NaN A 47.5 47.5 0 1 0 50 2.5" <----> jquery.js:6990
Error: Invalid value for <path> attribute d="M NaN NaN A 47.5 47.5 0 1 0 50 2.5" <----> jquery.js:6990

```html
<div round-progress
                    max="100"
                    current="vm.avg.count"
                    color="#45ccce"
                    bgcolor="#eaeaea"
                    radius="50"
                    stroke="5"
                    iterations="50"
                    animation="easeInOutQuart">
</div>
```

Any idea?
FYI : I'm using a ControllerAs 'vm'

**Comment by crisbeto** [comment]
I'm guessing that the `vm.avg.count` is `undefined` initially and its showing those NaNs. I'll have to look at a better way of handling it in the directive.

**Comment by crisbeto** [comment]
Its fixed in [0.2.12](https://github.com/crisbeto/angular-svg-round-progressbar/releases/tag/0.2.12). It will stay at 0 until it gets a value.
